db/migrate/20221103112116_add_prices_to_site_settings.rb

```diff
@@ -0,0 +1,11 @@
+class AddPricesToSiteSettings < ActiveRecord::Migration[6.0]
+  def change
+    add_column(:site_settings, :prices, :jsonb)
+    SiteSettings.current.update!(
+      electricity_price: BenchmarkMetrics::ELECTRICITY_PRICE,
```

Comment:
If this is merged before the analytics, then there won't be a price here?

Reply:
indeed.  i'll hard code these values instead as they'll be over-writable anyway.  :+1: